definitions/procedures/packages/check_for_reboot.rb

```diff
@@ -0,0 +1,14 @@
+module Procedures::Packages
+  class CheckForReboot < ForemanMaintain::Procedure
+    metadata do
+      description 'Check if system needs reboot'
+    end
+
+    def run
+      _status, output = execute_with_status('dnf needs-restarting --reboothint')
+      puts output
```

Comment:
if a reboot is required, _status will be 1 (instead of 0), can we mark the step as "warn" instead of "OK" then?

```suggestion
      status, output = execute_with_status('dnf needs-restarting --reboothint')
      if status != 0:
        set_warn(output)
      else
        puts output
      end
```